examples/acquiring_data/searching_multiple_wavelengths.py

```diff
@@ -0,0 +1,42 @@
+"""
+==============================================
+Searching for multiple wavelengths with Fido
+==============================================
+This example shows how you can search for several wavelengths
+of AIA data with Fido.
+"""
+from astropy import units as u
+
+from sunpy.net import Fido, attr
+from sunpy.net import attrs as a
+
+###############################################################################
+# Here we are demonstrating how you can search for specific wavelengths of
+# AIA data using `Fido` and the `sunpy.net.attr.AttrOr` function.
+# For example, you may only want a single wavelength, say 171 Angstrom.:
+
+aia_search = Fido.search(a.Time("2022-02-20 00:00", "2022-02-20 00:01"),
+                         a.Instrument("AIA"),
+                         a.Wavelength(171*u.angstrom))
+
+aia_search
```

Comment:
```suggestion
print(aia_search)
```
I think this needs to be printed to ensure it always renders.